bucket/rom-properties-np.json

```
@@ -0,0 +1,62 @@
+{
+    "version": "1.9",
+    "description": "This shell extension adds a few nice features to file browsers for managing video game ROM and disc images.",
+    "homepage": "https://github.com/GerbilSoft/rom-properties",
+    "license": "GPL-2.0",
+    "url": "https://github.com/GerbilSoft/rom-properties/releases/download/v1.9/rom-properties_1.9-windows.zip",
+    "hash": "f8a788700034b52bf007f58eec595e49892452366e72cf8ded68d8f9cb9dabdd",
+    "pre_install": [
+        "if (!(is_admin)) {",
+        "    error 'Administrator rights are required to install $app.'",
+        "    exit 1",
+        "}"
+    ],
+    "installer": {
+        "script": [
+            "if ($architecture -eq '64bit') {",
+            "    regsvr32 /s \"$dir\\amd64\\rom-properties.dll\"",
+            "}",
+            "regsvr32 /s \"$dir\\i386\\rom-properties.dll\"",
+            "taskkill /f /im explorer.exe | Out-Null"
+        ]
+    },
+    "post_install": "explorer",
```

```suggestion
    "installer": {
        "script": [
            "if ($architecture -eq '64bit') {",
            "    regsvr32 /s \"$dir\\amd64\\rom-properties.dll\"",
            "}",
            "regsvr32 /s \"$dir\\i386\\rom-properties.dll\"",
            "taskkill /f /im explorer.exe | Out-Null",
            "Start-Sleep -Seconds 3",
            "explorer"
        ]
    },
```

I guess you want to terminate and restart **explorer**. In this case, just use `Start-Sleep` to add a delay. Restarting **explorer** in `installer` or `post_install` actually doesn't make any difference.